soda/cmd/schema/load.go

```diff
@@ -17,14 +18,20 @@ var LoadCmd = &cobra.Command{
 	Use:   "load",
 	Short: "Load a schema.sql file into a database",
 	RunE: func(cmd *cobra.Command, args []string) error {
+		env := cmd.Flag("env")
+		if env == nil {
```

Comment:
hi @stanislas-m  ,
does this mean schema load will always requires --env flag?

Reply:
No, it's using the value provided by the root command. So it defaults to "development".
The only case it can be null, is if you try to use this specific command alone from another package (outside pop).

Reply:
Thanks for the review @u007! So no, it's using the value provided by the root command. It defaults to "development".
The only case it can be null, is if you try to use this specific command alone from another package (outside pop).

Reply:
i think its good :)